README.md

```diff
@@ -36,16 +36,22 @@ The XML schema, mapping rules and examples are available at [schemas/xml/](schem
 
 ### XMI
 
-The XMI file for the UML metamodel [schemas/xmi/](schemas/xmi/).
+The XMI file for the UML metamodel is available at [schemas/xmi/](schemas/xmi/).
 
 ### YAML
 
-The YAML file for the API Definition [schemas/yaml/](schemas/yaml/).
+The YAML files for the API definitions are available at [schemas/yaml/](schemas/yaml/).
 
+  
 ## Releases
 
 Currently, the following releases are available (see also the [releases](https://github.com/admin-shell-io/aas-specs/releases) section of this repository):
-* [3.0.RC02](https://github.com/admin-shell-io/aas-specs/releases/tag/v3.0.2.RC01) is a draft of the pre-release of `3.0.RC02`
+* [3.0.5.RC02](https://github.com/admin-shell-io/aas-specs/releases/tag/v3.0.5.RC02) is the corresponding release for the `V3.0RC02` version of the AAS metamodel, containing the normative schemas for the published document "Part 1: Details of the Asset Administration Shell - The exchange of information between partners in the value chain of Industrie 4.0 - **Version 3.0RC02**". *Use this release if you want to work with the latest specified AAS version.*
+* [3.0.4.RC01](https://github.com/admin-shell-io/aas-specs/releases/tag/v3.0.4.RC01) is the final release for the `V3.0RC01` version of the AAS metamodel, containing the normative schemas for the published document "Part 1: Details of the Asset Administration Shell - The exchange of information between partners in the value chain of Industrie 4.0 - **Version 3.0RC01**". *No additional bugfix or patch releases for the metamodel version `V3.0RC01` shall be expected.*
+* [3.0.3.RC01](https://github.com/admin-shell-io/aas-specs/releases/tag/v3.0.3.RC01) is a pre-release of the schemas for the published document "Part 1: Details of the Asset Administration Shell - The exchange of information between partners in the value chain of Industrie 4.0 - Version 3.0RC01" that solves several typos and bugs of `3.0.2.RC01`
```

Comment:
evtl. reichen auch nur die 3.0.4.RC01 und 3.0.5.RC02 da die anderen ja nicht mehr verwendet werden sollen, aber falsch ist es natürlich nicht

I miss chapter on Versioning schema:

We use the following versioning scheme:
<major version>.<minor version>.<bugfix version>.RC<number>

<major version> and <minor version> as well as RC<number> are identical to the UML specification document. Patches on the schemas are denoted by the <bugfix version>. 
RC stands for "Release Candidate", there can be more than one release candidate per UML specification version. If the UML specification document itself has a bugfix version then this is denotes by
<major version>.<minor version>.<bugfix version>.RC<number>-patch<Bugfix version of UML specification>